### DISCUSSION TITLE
Manipulating contributions by adjusting OS timezone

### DISCUSSION THREAD

**Discussion by nathanelcorpuz** [community/community#40669] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello, not sure where to report this, but I was recently testing timezones locally which led me to change date and time of my OS. 

I accidentally pushed to github while doing this, where I set my date to Dec 28, 2022 (it's currently Dec 3, 2022), and my profile reflects the wrong date.

![image](https://user-images.githubusercontent.com/77599339/205425805-e24a2291-36a7-4ff4-b224-78882a8ea266.png)

**Comment by radar-blip** [comment]
I'm not seeing it here, maybe its only reflected on your side.
Course, its not the 28th here yet. 😆 

![contros](https://user-images.githubusercontent.com/39196767/205426040-2b4ce868-1d1b-4013-9a73-bf8c00a1024b.png)

**Reply by nathanelcorpuz** [reply]
Maybe try switching to 2021 then back to 2022 view. That made my profile show further dates. Please let me know if that made you see it.

**Reply by radar-blip** [reply]
Yep. 

![contris](https://user-images.githubusercontent.com/39196767/205465379-2cfccf09-b13e-4e82-a726-d7e706dc0cf9.png)

**Comment by DSDmark** [comment]
The same happen to me. I was things to report it. The GitHub timezone sync with our system time.

**Comment by airtower-luna** [comment, marked as answer]
That's no bug. You don't even need to change the clock on your computer, `git commit` has a [`--date` option](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---dateltdategt) you can use to change the author date recorded in the commit, and it allows you to set author date *and* committer date as you like using the [`GIT_AUTHOR_DATE` and `GIT_COMMITTER_DATE` environment variables](https://git-scm.com/docs/git-commit#_commit_information).

**Comment by Seetheeda** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> Hello, not sure where to report this, but I was recently testing timezones locally which led me to change date and time of my OS. 
> 
> I accidentally pushed to github while doing this, where I set my date to Dec 28, 2022 (it's currently Dec 3, 2022), and my profile reflects the wrong date.
> 
> ![image](https://user-images.githubusercontent.com/77599339/205425805-e24a2291-36a7-4ff4-b224-78882a8ea266.png)

Dec 28 2022
Dec 2 2022

**Reply by Seetheeda** [reply]
Date  Dec  28 2022
Currently Dec 03 2022
